open_fdd/quality.py

```
@@ -185,9 +185,30 @@ def normalize_role_series(
     valid = valid & ~nulls
 
     if family in {"status", "command"} or family in STATUS_ROLES or family in COMMAND_ROLES:
-        # Status/command: 0/1 (and 0–100 cmd) are legitimate. Only sentinels/nulls fail.
+        # Status/command: 0/1 (and 0–100 cmd) are legitimate. Occupancy calendars
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)